content/02.introduction.md

```diff
@@ -121,7 +121,24 @@ The mechanism behind these insults has been suggested to be related to inflammat
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
+#### Subpopulations of Special Concern  
+
+The prevalence of different clinical features of COVID-19 infection differ in different areas of the world, presumably due to differential reporting and/or different levels of pre-existing risk factors [@doi:10.1007/s00592-020-01546-0].
```

Comment:
I think "pre-existing" may be more of an insurance term, you could just say "risk factors"

Reply:
I would write ..... "varying levels of pre-exitisting co-morbidities, such as hypertension, diabetes and lung diseases".

Reply:
also to check at some point: how big effect age structure / age distribution may have ... this is also more complicated b/c depends on other possible age-related differences and also differences in age contact structure / interactions between different age groups.

Reply:
I hadn't thought about age as a confounder -- I will check the reference you suggested.  Thanks!